request/auth.php

```diff
@@ -14,7 +14,7 @@
                     case 'login':
                         if (!empty($_POST['email']) && !empty($_POST['password'])) {
                             if (FirstAide\Utils::isValidEmail($_POST['email'])) {
-                                $Auth = FirstAide\Authentication::withEmailPassword(
+                                $Auth = FirstAide\Authentication::createInstanceWithEmailPassword(
```

Comment:
@npari I read about this, auth.php is a wrapper to handle global variables (such as$ _SESSION and $_POST) and the suggested approach to use it is outside the methods because methods will require access to global variables. It is never suggested to use global variables inside methods and classes. 
https://tomnomnom.com/posts/why-global-state-is-the-devil-and-how-to-avoid-using-it This explains it clearly.